packages/opentelemetry-sdk-trace-web/package.json

```diff
@@ -13,8 +13,8 @@
     "clean": "tsc --build --clean tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
     "lint": "eslint . --ext .ts",
     "lint:fix": "eslint . --ext .ts --fix",
-    "codecov:browser": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
-    "codecov:webworker": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+
```

Comment:
I guess the replacement left these blank lines

Reply:
good catch :+1:

Reply:
fixed in [d95e18c](https://github.com/open-telemetry/opentelemetry-js/pull/5011/commits/d95e18cd312d38114a542664df4b62ef40dedf69)